backend/slackbot/handler.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+const hook = "https://hooks.slack.com/services/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"
+
+const region = process.env.DEPLOY_REGION;
+const stage = process.env.STAGE;
+const gatewayURL = process.env.GW_URL;
+
+const axios = require('axios');
+
+module.exports = {
+    handler,
+    getFordApps,
+    getFordStatistics,
+    sendStats
+}
+
+async function handler () {
+    let fordApps = {};
+    let statistics = [];
+
+    // Get list of apps we want statistics for
+    try {
+        fordApps = await getFordApps();
+    } catch (error) {
+        return {
+            statusCode: 500, // Internal Server Error
+            error: `Error getting ford apps: ${error}`
+        };
+    }
+
+    // Get the statistics for those apps
+    try {
+        statistics = await getFordStatistics(fordApps);
+    } catch (error) {
+        return {
+            statusCode: 500,
+            error: `Error getting ford stats: ${error}`
+        };
+    }
+
+    try {
+        await sendStats(statistics);
+    } catch (error) {
+        return {
+            statusCode: 500,
+            error: `Error sending to slack: ${error}`
+        };
+    }
+
+    return {
+        statusCode: 200, // OK
+        body: JSON.stringify({
+            apps: statistics
+        })
+    };
+}
+
+/**
+ *
+ * @param gatewayURL the gateway url for aws lambda endpoint
+ * @returns returns list of ford apps
+ */
+async function getFordApps() {
+    let endpoint = gatewayURL + '/apps';
+    return axios.get(endpoint);
+
+}
+
+/**
+ *
+ * @param fordApps the list of apps we want statistics from
+ * @returns appStats a list of statistics for each specified version of an app
+ */
+async function getFordStatistics(fordApps){
+    let promises = [];  // Array for promises
+    let appStats = [];  // Array for App Statistics for each version of each app we want
+
+    let apps = fordApps.data;
+    let endpoint = gatewayURL + '/stats';
+
+    for (let i in apps){
+        let app = apps[i];
```

Comment:
Give `i` a more descriptive name - does this variable represent the appId, the index...?